src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -310,6 +312,37 @@ public Response deleteAuthenticatedUser(@PathParam("identifier") String identifi
 		}
 		return error(Response.Status.BAD_REQUEST, "User " + identifier + " not found.");
 	}
+        
+        @PUT
+        @Path("authenticatedUsers/changeIdentifier/{identifier}")
+        public Response changeAuthenticatedUserIdentifier(@PathParam("identifier") String oldIdentifier, String newIdentifier) {
+
+            if(null == oldIdentifier || oldIdentifier.isEmpty()) {
+                return error(Response.Status.BAD_REQUEST, "Old identifier provided to change is empty.");
+            } else if(null == newIdentifier || newIdentifier.isEmpty()) {
+                return error(Response.Status.BAD_REQUEST, "New identifier provided to change is empty.");
+            }
+            
+            AuthenticatedUser authenticatedUser = authSvc.getAuthenticatedUser(oldIdentifier);
+            if (authenticatedUser == null) {
+                return error(Response.Status.BAD_REQUEST, "User " + oldIdentifier + " not found in AuthenticatedUser");
+            }
+            
+            BuiltinUser builtinUser = builtinUserService.findByUserName(authenticatedUser.getUserIdentifier());
```

Comment:
do we need to check builtinuser? i.e. I could imagine a scenario where we would want to change the username for a Shib or Github user.

Reply:
so after discussion, let's move this get into the command (with the role assignment gets)